### PR TITLE
1955 static page edit - only premium & privileged users can edit static pages.

### PR DESCRIPTION
**Pull request by Hess3000**
#### Description
Added a helper for method: user_can_edit?

#### Motivation and Context
Allowing anyone to edit static pages is becoming a pain.

#### How Has This Been Tested?
Has not been tested.
Rails has been upgraded so tests do not run.

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by nisevi** [comment]
@Hess3000 I just merged in the latest changes from `develop` branch. Now you just need to do `git pull origin 1955_static_page_edit` and your branch should be up to date with that :+1: . Please, let me know if you encounter any issues while doing it :+1:

**Comment by nisevi** [comment]
Hi there @Hess3000, regarding the files that were added on `features/support/fixtures/cassettes` you maybe want to take a read to [this](https://github.com/AgileVentures/WebsiteOne/blob/develop/CONTRIBUTING.md#acceptance-tests-and-caching) :+1:

**Comment by nisevi** [comment]
@Hess3000 please let me know if you need help reverting changes for removing the files that were added here `features/support/fixtures/cassettes` :+1:

**Comment by nisevi** [comment]
Hey @Hess3000 I just mention it because its common when you start working on this repository, to get confused about the VCR files that get generated when you run the cucumber tests, if that is not your case just disregard my message :+1:

File: app/helpers/static_pages_helper.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+module StaticPagesHelper
+  
+  def user_can_edit?
+    current_user && current_user.is_privileged? || current_user.membership_type != 'Basic'
```
Comment (tansaku) [inline app/helpers/static_pages_helper.rb]: do we need the user_signed_in? ? I guess not - current_user nil check will handle that ...

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -149,4 +149,8 @@ def default_meta_description
   def present(model)
     yield(model.presenter)
   end
+
+  def user_can_edit?
```
Comment (tansaku) [inline app/helpers/application_helper.rb]: do we need this both in application helper and in static pages helper?

Comment (Hess3000) [reply app/helpers/application_helper.rb]: Would this be best to leave in application_helper so we can use it everywhere?

**Comment by tansaku** [comment]
Hi @Hess3000 - we've deployed the new content model that allows editing via github pull request so this feature is no longer needed.  Thanks for looking at it - hope we can find something else fun for you to work on :-)

**Pull request closed without merging**